### PR TITLE
doc: add todo list and image to readme

### PR DESCRIPTION
**Pull request by neutrinoks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +17,17 @@ For more information and to learn more about how to work with this pallet, the f
 - [Testing](#testing)
 - [Benchmarking](#benchmarking)
 
+### Open / Planned Points
+
+- Refine gas tuning and gas-weight logic
+- Update to latest Polkadot-SDK version in template node / pallet-move
```
Comment (asmie) [inline README.md:23]:
```suggestion
- Update to the latest Polkadot-SDK version in template node / pallet-move
```

File: README.md [hunk 2]
```diff
@@ -15,6 +17,17 @@ For more information and to learn more about how to work with this pallet, the f
...
+- Add more pallet tests
+- Develop full benchmarks
+- Multi-signature support
+- Protection mechanism for move address 0x1
```
Comment (asmie) [inline README.md:27]:
```suggestion
- Protection mechanism for Move address 0x1
```

**Review by asmie — APPROVED** [review]
:+1:

File: README.md (hunk 2 above)
Comment (neutrinoks) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (neutrinoks) [reply README.md]: Done

**Review by Rqnsom — APPROVED** [review]

**Pull request merged**